oauth2-server/src/test/java/com/clouway/oauth2/IssueNewTokenForClientTest.java

```diff
@@ -52,7 +51,7 @@ public void happyPath() throws IOException {
       will(returnValue(Optional.of(new Authorization("", "", "::user_id::", "::auth_code::", Collections.<String>emptySet(), Collections.singleton("::redirect_uri::")))));
 
       oneOf(tokens).issueToken(GrantType.AUTHORIZATION_CODE, client, "::user_id::", Collections.<String>emptySet(), anyTime);
-      will(returnValue(new TokenResponse(true, aNewToken().withValue("::token::").build(), "::refresh token::")));
+      will(returnValue(new TokenResponse(true, aNewToken().withValue("::token::").build(), "::refresh token::", "encodedIdToken")));
```

Comment:
"encodedToken" renamed to "::id token::" to follow existing testing pattern code.